src/storage.js

```diff
@@ -0,0 +1,32 @@
+export default class Store {
+  static getToDos = () => {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+    return tasks;
+  }
+
+  static addNewTask = (task) => {
+    const tasks = Store.getToDos();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  };
+
+  static clearInput = () => {
+    document.querySelector('#add-list').value = '';
+  };
+
+   static removeTask = (element) => {
+     let tasks = Store.getToDos();
+     tasks = tasks.filter((item) => item.index.toString() !== element.id.toString());
+     let i = 1;
+     tasks.forEach((task) => {
+       task.index = i;
+       i += 1;
+     });
+     localStorage.setItem('tasks', JSON.stringify(tasks));
+   };
+}
```

Comment:
- [ ] Well don implementing most of the tasks as requested. But for your remove task functionality, it seems there is a conflict with the `index rearrangement`. as you can see after removing the task starting from the top, it persisted on reload. Please address this.

https://user-images.githubusercontent.com/69077061/221183973-5d1b0279-e661-4c11-99ab-b9953268a62a.mp4

Reply:
Hi thank you for the review, may you please let me know what I need to do I don't see the error on my browser, and I don't understand what I need to fix.

Reply:
Dear @SaneMyburg, As you can see in my screenshot, I delete tasks but when I reload it appears again. This implies your list is not getting cleaned up on the first round. this only happens when I delete tasks starting from the top. Please notice how your task are still in the dom when I reload the page. `Every change should be saved into the local storage` which means when I reload the page I should not see the task I have already deleted. I hop this clarifies for you.

Reply:
Oh yes thank you, I get it now.